i18n-util/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "i18n-util",
+  "version": "1.0.0",
+  "description": "locale util",
+  "author": "nikhil.venkatraman@okta.com",
+  "license": "MIT",
+  "scripts": {
+    "test": "yarn jest",
+    "lint": "eslint .",
+    "clean-dist": "rimraf dist && rimraf ../i18n-util.bundle.*.js",
+    "build": " yarn clean-dist &&  webpack && cp dist/okta_help.htm ../ && cp dist/i18n-util.bundle.*.js ../"
+  },
+  "devDependencies": {
+    "eslint": "^7.10.0",
+    "html-webpack-plugin": "^4.5.0",
+    "rimraf": "^2.6.1",
+    "webpack": "^5.10.1",
+    "webpack-cli": "^4.2.0",
+    "jest": "24.9.0",
+    "jest-junit": "5.0.0"
+  },
+  "engines": {
+    "yarn": "^1.21.1",
+    "node": ">=8.11.1"
```

Comment:
I'm guessing we selected this version to match `okta-ui`, but would be surprised if Node 8 or 10 versions are supported by our dependencies. Can we pin this to `12.x`?

Reply:
yeah, updated to locale version of node "node": "12.13.0"